betterj.go

```diff
@@ -0,0 +1,35 @@
+package betterj
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+//I used the bytes.buffer because its dynamic and works good with compact and indent
+
+func MinifyJ(j string) (string, error) {
+	var buffer bytes.Buffer
+
+	//json string converted to bytes for the compact function
```

Comment:
```suggestion
	// json string converted to bytes for the compact function
```

And all other comments also have a space between `//` and `text of comment`.

Reply:
Will rectify it